src/main/drivers/bus_i2c_hal.c

```diff
@@ -56,13 +56,18 @@ static void i2cUnstick(IO_t scl, IO_t sda);
 #define GPIO_AF4_I2C GPIO_AF4_I2C1
 
 const i2cHardware_t i2cHardware[I2CDEV_COUNT] = {
+#if defined(STM32F7)
 #ifdef USE_I2C_DEVICE_1
     {
         .device = I2CDEV_1,
         .reg = I2C1,
         .sclPins = { I2CPINDEF(PB6), I2CPINDEF(PB8) },
         .sdaPins = { I2CPINDEF(PB7), I2CPINDEF(PB9) },
+#ifdef STM32H7
```

Comment:
Unless I am overlooking something crucial here, these conditionals are not needed, as the `#else` will be taken every time due to the outer `#if defined(STM32F7)`, and the `#ifdef STM32H7` will be taken every time for the `#elif defined(STM32H7)` case below.

Reply:
Doh, you are right. I probably first started out with modifying the F7 block, then realized that H7's pin are scattered across multiple AFs and created the dedicated block for H7, and forgot to revert the F7 block.